src/sonic-ctrmgrd/ctrmgr/container

```diff
@@ -301,19 +311,27 @@ def container_kill(feature, **kwargs):
 
     init()
 
-    set_owner, _ = read_config(feature)
+    ret = SUCCESS
+    set_owner, _ , state = read_config(feature)
     current_owner, remote_state, _ = read_state(feature)
     docker_id = container_id(feature)
     remove_label = (set_owner != "local") or (current_owner != "local")
 
-    debug_msg("{}: set_owner:{} current_owner:{} remote_state:{} docker_id:{}".format(
-        feature, set_owner, current_owner, remote_state, docker_id))
+    print("{}: set_owner:{} current_owner:{} remote_state:{} docker_id:{} state:{}".format(
+        feature, set_owner, current_owner, remote_state, docker_id, state))
 
+    print(remove_label)
```

Comment:
Maybe print is left after some debugging?

Reply:
Done